src/openloco/map/tilemgr.cpp

```diff
@@ -200,12 +201,10 @@ namespace openloco::map::tilemgr
     }
 
     // 0x004CBE5F
-    void map_invalidate_tile_full(uint16_t x, uint16_t y)
+    void map_invalidate_tile_full(coord_t x, coord_t y)
     {
-        registers regs;
-        regs.ax = x;
-        regs.cx = y;
-        call(0x004CBE5F, regs);
+        auto pos = map::map_pos(x, y);
+        ui::viewportmgr::invalidate(pos, 0, 1088, ZoomLevel::eighth);
```

Comment:
where are you getting 1088 from. My reading of the assembly has it that it would be 1120 or (16 * 70)

Reply:
Yes, you are right, I don't know why is in the hook code 1088 then. Going to change it.

Reply:
This one? https://github.com/OpenLoco/OpenLoco/blob/9f595a51f504462240916267b0a1300eaf5611eb/src/openloco/viewportmgr.cpp#L381-L387

Reply:
@AaronVanGeffen Yes, that one.